amundsen_application/api/utils/search_utils.py

```diff
@@ -25,6 +25,9 @@ def map_dashboard_result(result: Dict) -> Dict:
         'product': result.get('product', ''),
         'uri': result.get('uri', ''),
         'url': result.get('url', ''),
+        # TODO: Bookmark logic relies on key, opting to add this here to avoid messy logic in
+        # React app and we have to clean up later.
+        'key': result.get('uri', ''),
```

Comment:
I see what you mean. Unifying the usage of `key` or `uri` in metadata service would make this issue go away.